packages/fxa-settings/src/components/PageSettings/index.tsx

```diff
@@ -32,6 +32,14 @@ export const PageSettings = (_: RouteComponentProps) => {
         <Profile />
         <Security />
         <ConnectedServices />
+        <div className="flex mx-4 tablet:mx-0">
+          <a
+            className="cta-caution cta-base text-sm transition-standard mt-12"
+            href="/beta/settings/delete_account"
```

Comment:
It looks like at some point we started using a [constants file](https://github.com/mozilla/fxa/blob/delete-account-page/packages/fxa-settings/src/constants/index.tsx#L5:L5) for paths, you might consider continuing with that (or using a base path constant) since at some point we're going to need to remove `beta/` from all these paths.

Reply:
Would you be able to use the `Link` component to do the navigation?

Reply:
Oh ya that's a good point 👍